tests/monotouch-test/Foundation/NSExpressionTest.cs

```diff
@@ -45,13 +45,22 @@ void TestProperties (NSExpression expression, List<string> availableProperties)
 			}
 		}
 
-
-		[Test]
-		public void FromConstant ()
-		{
-			using (var expression = NSExpression.FromConstant (new NSString ("Foo")))
-			using (var result = expression.EvaluateWith (null, null) as NSString)
-				Assert.AreEqual ("Foo", result.ToString ());
+		[TestCase("Foo", Result = "Foo")]
+		[TestCase(null, Result = null)]
+		public object FromConstant (object input)
+		{
+            NSObject value = null;
+
+            switch (input)
+            {
+                case String stringValue:
+                    value = new NSString(stringValue);
+                    break;
+            }
+
+            using (var expression = NSExpression.FromConstant(value))
+            using (var result = expression.EvaluateWith(null, null) as NSObject)
+                return result?.ToString();
```

Comment:
Coding style: space before opening parenthesis (in the previous two lines as well).